packages/stark-ui/src/modules/pagination/components/pagination.component.ts

```diff
@@ -165,16 +165,22 @@ export class StarkPaginationComponent extends MatPaginator implements OnInit, On
 	public normalizePaginationConfig(config: StarkPaginationConfig): StarkPaginationConfig {
 		if (!config) {
 			// initialize paginationConfig to prevent errors in other functions depending on this config
-			this.logger.warn(componentName + ": No configuration defined in pagination component. TotalItems set to 0 by default");
+			this.logger.warn(componentName + ": No configuration defined in pagination component. TotalIt  ems set to 0 by default");
```

Comment:
Could you please put back this previous code ?
`this.logger.warn(componentName + ": No configuration defined in pagination component. TotalItems set to 0 by default");`